airbyte-config/init/src/main/resources/seed/source_specs.yaml

```diff
@@ -9889,6 +9889,15 @@
     supportsNormalization: false
     supportsDBT: false
     supported_destination_sync_modes: []
+- dockerImage: "airbyte/source-public-apis:0.1.0"
+  spec:
+    documentationUrl: https://api.publicapis.org
```

Comment:
@YiyangLi it was published with the wrong documentationUrl

Reply:
yep, changed it to `https://docs.airbyte.com/integrations/sources/public-apis`